lib/request.coffee

```diff
@@ -17,6 +17,7 @@ exports.request = (options = {}, callback, onProgress = _.noop) ->
 	_.defaults options,
 		method: 'GET'
 		gzip: true
+		onProgress: _.noop
```

Comment:
It would make more sense to default this in `utils.pipeRequest`, where it's actually used.